lambdas/sys_user_management/src/app.py

```diff
@@ -119,31 +117,26 @@ def delete_customer_managed_policies(self):
 
         for policy in policies_to_die:
             self.iam_client.delete_policy(PolicyName=policy)
-            logger.info("Deleted policy:", policy)
+            logger.info("Deleted policy: %s", policy)
             self.stats['policies_deleted'] += 1
-            self.stats['policies_deleted'].append(policy)
 
 
     def delete_iam_roles(self):
-
         response = self.iam_client.list_roles()
-        roles_to_die = response['Roles']
+        roles = [x for x in response['Roles'] if x not in self.config['cleanup_exclude_list_role']]
 
-        for role in roles_to_die:
+        for role in roles:
             role_name = role['RoleName']
-
-            if role_name != self.config['cleanup_exclude_list_role']:
-                self.iam_client.delete_role(RoleName=role_name)
-                logger.info('Role successfully deleted:', role_name)
-                self.stats['roles_deleted'] += 1
-                self.stats['roles_deleted'].append(role_name)
-            else:
-                logger.info('Role is in exclude list:', role_name)
+            self.iam_client.delete_role(RoleName=role_name)
+            logger.info('Role successfully deleted: %s', role_name)
+            self.stats['roles_deleted'] += 1
+        else:
```

Comment:
```
for i in _iterable_:
    pass
else:
    print("fires in case there was no 'break'")
```
Your message in the `else` statement will always fire since you do not break (and don't want to break) and can make some confusion. Thus it doesn't make much sense here. You better just remove it.